backend/api/v2beta1/swagger/kfp_api_single_file.swagger.json

```diff
@@ -1678,19 +1678,6 @@
         }
       }
     },
-    "protobufListValue": {
```

Comment:
Curious why do we see such deletion in generated code? Does that mean the last released api client doesn't really match our proto definition?

Reply:
I don't know. I also noticed that `PipelineSpec *ProtobufStruct` changed to `PipelineSpec interface{}`